integration_tests/CMakeLists.txt

```diff
@@ -351,7 +351,8 @@ RUN(NAME structs_11          LABELS cpython llvm c)
 RUN(NAME structs_12          LABELS cpython llvm c)
 RUN(NAME structs_13          LABELS llvm c
     EXTRAFILES structs_13b.c)
-RUN(NAME structs_14          LABELS cpython llvm c)
+RUN(NAME structs_14          LABELS llvm c
+    EXTRAFILES structs_14b.c)
```

Comment:
We might need to do a separate test, since we lost CPython test here. We want to ideally test everything via CPython, but at least what we were testing before.